README.md

```diff
@@ -2,6 +2,10 @@
 
 # Agoric Platform SDK
 
+[![build status](https://github.com/Agoric/SES-shim/workflows/CI/badge.svg?branch=master)](https://github.com/Agoric/SES-shim/actions?query=workflow%3ACI)
```

Comment:
_please fix_:

Looks like this is badging about the SES-shim repo not agoric-sdk

Reply:
good catch!